<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceptionistPageClass</class>
 <widget class="QMainWindow" name="ReceptionistPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReceptionistPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/GymPadelSystem/Resources/Gym.png&quot;);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>800</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						QPushButton
						{
						background: none;
						background-color: rgba(255, 51, 51,0.9);
						color: white;
						border-radius: 25px;
						}
						QPushButton:hover
						{
						border: 2px solid rgba(255, 51, 51,0.9);
						background-color: rgba(250, 250, 250,0.8);
						color: rgb(255, 51, 51);
						}
					</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QWidget" name="LWidget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>1351</width>
      <height>631</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						background: none;
						background-color: rgba(0,0,0,0.7);
						border-radius: 25px;
					</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>320</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background: none;
							font-size: 30px;
							color: rgba(135, 206, 235,0.8)
						</string>
     </property>
     <property name="text">
      <string>Receptionist Dashboard</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="SMTextBox">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background: none;
							background-color: rgba(250,250,250,0.7);
							padding-left: 15px;
							font-size: 15px;
							color: black;
							border-color:  rgba(135, 206, 235,0.8);
						</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Search Member</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchButton">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>150</y>
       <width>120</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							QPushButton
							{
							background: none;
							background-color: rgba(135, 206, 235,0.9);
							color: black;
							border-radius: 25px;
							}
							QPushButton:hover
							{
							border: 3px solid rgba(135, 206, 235,0.9);
							background-color: rgba(250, 250, 250,0.8);
							}
						</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLabel" name="IMLabel">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>230</y>
       <width>220</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background: none;
							font-size: 15px;
							background-color: rgba(250, 0, 0, 0.5);
							border-radius: 20px;
						</string>
     </property>
     <property name="text">
      <string>Invalid member</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>300</y>
       <width>1210</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							background: none;
							background-color:rgba(0,0,0,0.7) ;
							color:rgba(135, 206, 235,0.8);


						</string>
     </property>
     <property name="rowCount">
      <number>1</number>
     </property>
     <property name="columnCount">
      <number>12</number>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>First Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Middle Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Last Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visits</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Day</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Month</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Year</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Day</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Month</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End Year</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Plan</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
